gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/kubernetes/KubernetesSyncService.java

```diff
@@ -122,8 +122,12 @@ private Completable handleConfigMapEvent(Event<ConfigMap> kubEvent) {
                 switch (kubEvent.getType()) {
                     case "ADDED":
                     case "MODIFIED":
-                        event.setType(EventType.PUBLISH_API);
-                        api.setLifecycleState(LifecycleState.STARTED);
+                        api.setLifecycleState(LifecycleState.valueOf(apiDefinition.getState()));
```

Comment:
To manage state, I suggest to store the state in another attribute of the configmap so the definition remains untouched, ex:  `String state = configMap.getData().get(DATA_STATE);`